build/__tools__/run-tests.sh

```diff
@@ -51,199 +45,11 @@ function cleanup {
 
 trap cleanup EXIT
 
-# -----------------------------------------------------------------
-# some checks to make sure we are ready to run
-# -----------------------------------------------------------------
-if [ "${PDO_LEDGER_TYPE}" == "ccf" ]; then
-    if [ ! -f "${PDO_LEDGER_KEY_ROOT}/networkcert.pem" ]; then
-        die "CCF ledger keys are missing, please copy and try again"
-    fi
-fi
-
-# -----------------------------------------------------------------
-yell run unit tests for python, common, contracts and eservice
-# -----------------------------------------------------------------
-say run unit tests for python package
-cd ${SRCDIR}/python
-try make TEST_LOG_LEVEL=${PDO_LOG_LEVEL} test > /dev/null
-
-say run unit tests for common library
-cd ${SRCDIR}/common/build
-try make TEST_LOG_LEVEL=${PDO_LOG_LEVEL} test > /dev/null
-
-say run unit tests for eservice
-cd ${SRCDIR}/eservice
-try make TEST_LOG_LEVEL=${PDO_LOG_LEVEL} test > /dev/null
-
-say run unit tests for contracts
-cd ${SRCDIR}/contracts
-try make TEST_LOG_LEVEL=${PDO_LOG_LEVEL} test > /dev/null
-
-# -----------------------------------------------------------------
-yell start enclave and provisioning services
-# -----------------------------------------------------------------
 try ${PDO_HOME}/bin/ss-start.sh --loglevel ${PDO_LOG_LEVEL}
 try ${PDO_HOME}/bin/ps-start.sh --loglevel ${PDO_LOG_LEVEL} --ledger ${PDO_LEDGER_URL} --clean
 try ${PDO_HOME}/bin/es-start.sh --loglevel ${PDO_LOG_LEVEL} --ledger ${PDO_LEDGER_URL} --clean
```

Comment:
Nice separation, but... is there any reason for not moving even these `*s-start` lines (and the cleanup) to service-test.sh?